pkg/daemon/discover/discover.go

```diff
@@ -372,6 +395,18 @@ func probeDevices(context *clusterd.Context) ([]sys.LocalDisk, error) {
 		device.Filesystem = fs
 		device.Empty = clusterd.GetDeviceEmpty(device)
 
+		// Add the information provided by ceph-volume inventory
+		if cvInventory != nil {
+			cvDeviceData := clusterd.CephVolumeDeviceData(device.Name, cvInventory)
+			if cvDeviceData != nil {
+				device.CephVolumeData = cvDeviceData
+			} else {
+				logger.Errorf("Ceph volume information for device %s not found", device.Name)
+			}
+		} else {
+			device.CephVolumeData = nil //sys.CephVolumeDeviceData{}
```

Comment:
What's the comment about?

Reply:
I forget to remove it. Thanks!. Now deleted.